server/app.js

```diff
@@ -0,0 +1,38 @@
+const config = require('./config/config')
```

Comment:
You are using `require` vs es6 `import` statements. Does this mean we would not be using es6 for the project?

Reply:
Oh I see, is there an import support now (in node)? (I don't check up on these details). we definitely are using es6 but for the backend i'm used to using require since "import" support has not been included in node. However all client codes use import and all es6 > statements since it includes babel

Reply:
Yeah. We can use `import` statement for the back-end also. It means we will need to also install babel. For development, we will not necessarily need to build/trans-pile to `es5` using babel but before we deploy to production we will have to. So our `npm` scripts can have the necessary commands for those.

Reply:
> Not meant for production use

You should not be using babel-node in production. It is unnecessarily heavy, with high memory usage due to the cache being stored in memory. You will also always experience a startup performance penalty as the entire app needs to be compiled on the fly.

Check out the example Node.js server with Babel for an idea of how to use Babel in a production deployment.

> 

> Yeah. We can use `import` statement for the back-end also. It means we will need to also install babel. For development, we will not necessarily need to build/trans-pile to `es5` using babel but before we deploy to production we will have to. So our `npm` scripts can have the necessary commands for those.

what do you think about this warning on babel (for node)  https://babeljs.io/docs/en/babel-node